index.html

```diff
@@ -5,22 +5,58 @@
     	<meta name="description" content="Your description goes here">
     	<meta name="keywords" content="one, two, three">
 
-		<title>Blank Template</title>
-
+		<title>Relaxr</title>
+		<!-- Google fonts -->
+		<link href='https://fonts.googleapis.com/css?family=Open+Sans:400,300,300italic,600,800' rel='stylesheet' type='text/css'>
 		<!-- external CSS link -->
-		<link rel="stylesheet" href="css/reset.css">
+		<link rel="stylesheet" href="css/normalize.css">
 		<link rel="stylesheet" href="css/style.css">
 
 	</head>
 	<body>
```

Comment:
@qodesmith guess I didn't read the design file correctly i assumed the page was a full width landing page I will made adjustments to the Stylesheet so that it follows the design specs. Thanks for looking over my work man